packages/lit-node-client/src/lib/lit-node-client.ts

```diff
@@ -80,15 +80,18 @@ import {
 
 import { computeAddress } from '@ethersproject/transactions';
 import { SiweMessage } from 'lit-siwe';
-import { joinSignature } from 'ethers/lib/utils';
+import { joinSignature, sha256 } from 'ethers/lib/utils';
 import {
   checkAndSignAuthMessage,
   getSessionKeyUri,
   parseResource,
 } from '@lit-protocol/auth-browser';
+import { importer } from 'ipfs-unixfs-importer';
```

Comment:
@Ansonhkg this seems to be producing an error when i try to run the tests.  Can you run `yarn test:packages` and see if the error is something you're familiar with?

Reply:
It seems like Jest isn't transforming those modules for us, but the libraries we are using ships a minified version of their code, so we can just use the in the jest tests and skip transforming all together.

`jest.config.ts`

```
  moduleNameMapper: {
    '^ipfs-unixfs-importer':
      'node_modules/ipfs-unixfs-importer/dist/index.min.js',
    '^blockstore-core': 'node_modules/blockstore-core/dist/index.min.js',
  },
```

Reply:
nice find